monai/fl/utils/constants.py

```diff
@@ -9,38 +9,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from monai.utils.enums import StrEnum
 
-class WeightType:
+
+class WeightType(StrEnum):
     WEIGHTS = "fl_weights_full"
     WEIGHT_DIFF = "fl_weight_diff"
 
 
-class ExtraItems:
+class ExtraItems(StrEnum):
     ABORT = "fl_abort"
     MODEL_NAME = "fl_model_name"
     CLIENT_NAME = "fl_client_name"
+    APP_ROOT = "fl_app_root"
 
 
-class FlPhase:
+class FlPhase(StrEnum):
```

Comment:
Hi @wyli ,

Should we put all the `StrEnum` here to the `monai/utils/enums.py`?

Thanks.

Reply:
I think perhaps we revisit this after https://github.com/Project-MONAI/model-zoo/issues/52